modules/cookbooks/pages/rag.adoc

```diff
@@ -0,0 +1,212 @@
+= Retrieval-Augmented Generation (RAG)
+// tag::single-source[]
+:description: How to configure Redpanda Connect to create a RAG pipeline, using PostgreSQL and PGVector.
```

Comment:
Let's not single-source the description. We may want to customize it for cloud one day.

Reply:
Sorry what is the suggestion? Put the single-source tag below the description or `ifdef` the description? Sorry I'm not very familar with how all these steps get rendered.

Reply:
To put the // tag::single-source[] below the description.

Reply:
Done